utils/fasm2pips.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
what is this utility for?

Reply:
oh for highlighting in vivado

Reply:
It converts a FASM file into a Vivado tcl command that highlights the enabled pips from the FASM file.  Useful for sanity checking VPR routing output, etc.  I'm considering adding more logic to highlight upstream and downstream nodes from the pips, along with potentially highlight routing within the CLB sites.  For now it just highlights enabled routing pips.

Reply:
I'll add a docstring.